api/specs/webserver/openapi-projects-ports.yaml

```diff
@@ -20,7 +20,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Envelope_dict_uuid.UUID__simcore_service_webserver.projects.projects_ports_handlers.ProjectPortGet__'
+                $ref: '#/components/schemas/Envelope_dict_uuid.UUID__simcore_service_webserver.projects.projects_ports_handlers.ProjectInputGet__'
```

Comment:
Should not be ProjectInput**s**Get ?

Reply:
I'm confused with your usage of singular vs plurals here

Reply:
aha, ok. you got me confused for a moment but i think it is correct.  The reponse body is a ``list[ProjectInputGet]`` therefore it refers to a single item. See the signatures of the routes in 
https://github.com/ITISFoundation/osparc-simcore/blob/a7bd450846fc7c770067ac424cc020e40412aa94/api/specs/webserver/scripts/openapi_projects_ports.py#L33

There is more clear. The entire OAS is auto-generated with those